test/integration/other/migrations.js

```diff
@@ -866,3 +866,46 @@ describe('database migrations from 20230802: delete orphan submissions', functio
 
   }));
 });
+
+describe.skip('database migration: 20231002-01-add-conflict-details.js', function test() {
+  this.timeout(20000);
+
+  it('should update dataReceived and baseVersion of existing entity defs', testServiceFullTrx(async (service, container) => {
+    await upToMigration('20231002-01-add-conflict-details.js', false);
+    await populateUsers(container);
+    await populateForms(container);
+
+    const asAlice = await service.login('alice');
+
+    await asAlice.post('/v1/projects/1/forms?publish=true')
+      .set('Content-Type', 'application/xml')
+      .send(testData.forms.simpleEntity)
+      .expect(200);
+
+    const dsId = await container.oneFirst(sql`select id from datasets limit 1`);
+
+    // Create the entity in the DB directly because application code has changed.
+    const newEntity = await container.one(sql`
+    INSERT INTO entities (uuid, "datasetId", "creatorId", "createdAt")
+    VALUES (${uuid()}, ${dsId}, 5, now())
+    RETURNING "id"`);
+
+    // Create two entity defs
+    await container.run(sql`
+      INSERT INTO entity_defs ("entityId", "createdAt", "creatorId", "current", "label", "data")
+      VALUES (${newEntity.id}, now(), 5, true, 'some label', '{"foo": "bar"}')`);
```

Comment:
Seems like `current` should be `false` if another def is about to be created.

Reply:
it's not the best/most coherent entity def 😅 but at least `current` is not a relevant part of the migration.

Reply:
I will make these improvements in case anyone (future me?) decides to "borrow" this entity-insertion code in the future...